jekyll/_cci2/deployment_integrations.md

```diff
@@ -6,8 +6,33 @@ categories: [deploying]
 order: 10
 ---
 
+## Available Deployment Targets
 
-## Conditional Deployment 
+You can deploy to any service you choose with CircleCI 2.0. By adding commands to `.circleci/config.yml` and setting secrets on the 'Project Settings > Environment Variables' screen, you have a lot of flexibility. For deployment targets other than those listed in 'Deployment Integrations' below, please see the '<a href="#other-deployments">Other Deployments</a>' section.
+
+## Deployment Integrations
+
+**Note:** Currently (during Beta), the CircleCI 'Project Settings' UI has a section titled 'Continuous Deployment':
```

Comment:
Given that "Beta" is basically over, perhaps we should just go ahead and point out this is legacy?